packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -76,20 +78,19 @@ abstract class CameraPlatform extends PlatformInterface {
     throw UnimplementedError('prepareForVideoRecording() is not implemented.');
   }
 
-  /// Start a video recording and save the file to [path].
+  /// Starts a video recording
   ///
-  /// A path can for example be obtained using
-  /// [path_provider](https://pub.dartlang.org/packages/path_provider).
+  /// If no [file] parameter is provided, the recording will be saved to a new file in a temporary location.
```

Comment:
nit: wrap the documentation at 80 characters.
```suggestion
  /// If no [file] parameter is provided, the recording will be saved to a new 
  /// file in a temporary location.
```